mappers/bluetooth_mapper/README.md

```diff
@@ -0,0 +1,318 @@
+# Bluetooth Mapper
+
+
+## Introduction
+
+Mapper is an application that is used to connect and control devices. This is an implementation of mapper for 
+bluetooth protocol. The aim is to create an application through which users can easily operate devices using bluetooth protocol for communication to the KubeEdge platform. The user is required to provide the mapper with the information required to control their device through the configuration file these can be changed at runtime by providing the input through the MQTT broker.
```

Comment:
through the configuration file --> need  a period . 

these can be changed  -> These can be changed.

Reply:
done, thanks